### ISSUE TITLE
Menu type=overlay blocks user interactions after open and close

### ISSUE THREAD

**Issue by SevenInitiative** [driftyco/ionic2#835]
This bug can be seen in the menu component documentation when you switch to the Android (material design) preview (which defaults to using a menu with type=overlay)...

http://ionicframework.com/docs/v2/api/components/menu/MenuToggle

After opening and closing the menu once I noticed that the menu cannot be reopened (either by clicking and dragging from left to right or by clicking on the menuToggle button).

I had a look into the issue and think it just requires a styling fix. It actually appears that after closing the menu[type=overlay] all user interactions get blocked by the .backdrop element which is still being displayed over the top of the app (though invisible). Using Chrome tools I took the "display: block" property off of "ion-menu[type=overlay] .backdrop" and added it instead to "ion-menu[type=overlay] .backdrop.show-backdrop" selector and this seemed to solve the issue for me.

I'll commit the proposed fix on a new branch and make a pull request. Hope that's cool :)

**Comment by brandyscarney** [comment]
This may be a duplicate of https://github.com/driftyco/ionic2/issues/803

**Comment by SevenInitiative** [comment]
Ah ok, my bad - it does look like the same bug to me. I didn't think to look though closed issues.

**Comment by brandyscarney** [comment]
It's fine! It should be in the next release but there's a workaround in that issue until we release. :smile:

**Comment by brandyscarney** [comment]
We appreciate you wanting to make a PR to fix it though! I just didn't want you to waste your time if it's the same issue.

**Comment by SevenInitiative** [comment]
No problem at all. My css hack should suffice till I can update to the next release :)

In case anyone else comes across this and wants an easy temp fix, I've just added these rules to app.core.scss to force the menu backdrop to only display while the menu is open:

ion-menu[type=overlay] {
  .backdrop {
    display: none;
    &.show-backdrop {
      display: block;
    }
  }
}

**Comment by adamdbradley** [comment]
This issue should solve this problem: https://github.com/driftyco/ionic2/issues/803
